app/models/concerns/subscribable.rb

```diff
@@ -3,5 +3,13 @@ module Subscribable
 
   included do
     has_many :subscriptions, as: :subscribable, dependent: :destroy
+
+    after_create :create_subscription
```

Comment:
Why aren't we using this in `Comment`?

Reply:
Nor sure what you mean.
Yes we are? (https://github.com/dradis/dradis-ce/pull/265/files#diff-3b4cc13a815e6a32db590e41f5d00d49R7)
But `Comment` is not a `Subcribable`, so it does it itself, not with this concern.